### PR TITLE
tr(mixit): initialize project

### PR DESCRIPTION
**Pull request by cpuy**
2 commits

One for project initialization and one to add node_modules, will  be easier for us

### CODE REVIEW

**Comment by cpuy** [comment]
retest this please

**Comment by cpuy** [comment]
hello slack !!

File: js/hello-mixit.controller.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+        .module('hello-mixit', [])
+        .controller('HelloMixitCtrl', HelloMixitCtrl);
+
+    function HelloMixitCtrl() {
+        var vm = this;
+
+        vm.greetings = "Hello World !";
+
+    }
+
```
Comment (velcrin) [inline js/hello-mixit.controller.js:11]: Can we just use `this` and remove `vm` variable?
Can we also fix those unnecessary new lines?

Comment (cpuy) [inline js/hello-mixit.controller.js:11]: Yep Ill do that

**Pull request merged**